packages/manager/modules/hub/src/components/payment-status-tile/translations/Messages_fr_FR.json

```
@@ -0,0 +1,15 @@
+{
+  "ovh_manager_hub_payment_status_tile_title": "Derniers status de paiement des services",
```

```suggestion
  "ovh_manager_hub_payment_status_tile_title": "Derniers statuts du paiement des services",
```

Not sure about the `du` but I'm sure about the `statuts`

---

`de` doesn't sound strange to me, I'll check and correct if needed

---

Yes I really don't know about the `du` 🙂

---

@FredericEspiau I got confirmation that it was "de"